src/features/ajax/aggregate/index.js

```diff
@@ -96,7 +94,7 @@ export class Aggregate extends AggregateBase {
           handle(SUPPORTABILITY_METRIC_CHANNEL, ['Ajax/Events/Excluded/App'], undefined, FEATURE_NAMES.metrics, ee)
         }
         return
-      }
+      } else handle('bstXhrAgg', ['xhr', hash, params, metrics], undefined, FEATURE_NAMES.sessionTrace, ee)
```

Comment:
Should make this a separate if-check based off `block_internal` value, as stated, rather than relying on the deny-list which may include non-internal URLs .

Reply:
I guess what we are really after is that the denylist impacts the STN nodes.  if we are denying AJAX events we should deny STN as well, since this is pre-monetization

Reply:
I will revise the desc to better match that

Reply:
IIRC the last time we attempted to move these into behind the deny-list, it affected ingest and was reverted. Has John flip the light to green?

Reply:
Session traces are not currently billable -- we are currently just eating any costs associated with storing stn data -- so if anything this will actually reduce COGS. There's no better time to remove data than before it's actually billable.